### PR TITLE
K8s: expand PVC

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Preview: 
https://redis.io/docs/staging/DOC-3664/operate/kubernetes/re-clusters/expand-pvc/

### CODE REVIEW

**Review by almoggue — APPROVED** [review]

File: content/operate/kubernetes/re-clusters/expand-pvc.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Verify the StorageClass is configured with `allowVolumeExpansion: true`.
+- Verify your storage driver supports online expansion.
+
+### Not supported
+
+- Shrinking (reducing the size) of your PVC is not allowed. This process only allows you to expand (size up) your PVC.
+
+### Highly recommended
+
+- We highly recommend you backup your databases before beginning this PV expansion process.
```
Comment (dwdougherty) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:36]: Should PV be PVC?

```suggestion
- We highly recommend you backup your databases before beginning this PVC expansion process.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Just one comment. I just reviewed language, as I'm not qualified to judge the technical content.

**Review by dwdougherty — APPROVED** [review]
LGTM

File: content/operate/kubernetes/re-clusters/expand-pvc.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+categories:
+- docs
+- operate
+- kubernetes
+description: 
+linkTitle: Expand PVC
+weight: 15
+---
+
+PersistentVolumeClaims (PVC) are created by the Redis Enterprise for Kubernetes operator and use dy the RedisEnterpriseCluster (REC). PVCs are created with a specific size and can be expanded if the underlying storage class supports it.
```
Comment (mich-elle-luna) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:13]:
```suggestion
PersistentVolumeClaims (PVC) are created by the Redis Enterprise for Kubernetes operator and use the RedisEnterpriseCluster (REC). PVCs are created with a specific size and can be expanded if the underlying storage class supports it.
```

**Review by mich-elle-luna — CHANGES_REQUESTED** [review]
a typo I think?

File: content/operate/kubernetes/re-clusters/expand-pvc.md (hunk 2 above)
Comment (dwdougherty) [reply content/operate/kubernetes/re-clusters/expand-pvc.md:13]: Probably "used by" .

Comment (mich-elle-luna) [reply content/operate/kubernetes/re-clusters/expand-pvc.md:13]: Why are we referring to the 'operator and not 'you'?

Comment (mich-elle-luna) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:13]: I think the first sentence should describe this article and that it provides instructions for expanding PVCs by applying changes to the rec.yaml file then monitoring it.

File: content/operate/kubernetes/re-clusters/expand-pvc.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+Title: Expand PersistentVolumeClaim (PVC)
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description:
```
Comment (mich-elle-luna) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:8]: this should have a description

**Review by mich-elle-luna — CHANGES_REQUESTED** [review]
thank you! I think this needs some work, let me know if you need help with it.

File: content/operate/kubernetes/re-clusters/expand-pvc.md (hunk 2 above)
Comment (kaitlynmichael) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:13]:
```suggestion
PersistentVolumeClaims (PVC) are created by the Redis Enterprise for Kubernetes operator and used by the RedisEnterpriseCluster (REC). PVCs are created with a specific size and can be expanded if the underlying storage class supports it.
```

File: content/operate/kubernetes/re-clusters/expand-pvc.md (hunk 3 above)
Comment (kaitlynmichael) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:8]:
```suggestion
description: Expand your persistent volume claim with the Redis Enterprise for Kubernetes operator.
```

File: content/operate/kubernetes/re-clusters/expand-pvc.md (hunk 2 above)
Comment (kaitlynmichael) [reply content/operate/kubernetes/re-clusters/expand-pvc.md]: The operator is part of our Kubernetes software.

File: content/operate/kubernetes/re-clusters/expand-pvc.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  persistenceStatus:
+    status: Provisioned
+    succeeded: 3/3
+```
+
+### Troubleshooting
+
+If an error occurs during this process:
+
+- Examine the status and events of the REC and PVC objects.
```
[the remark this answers is not in the document]
Comment (kaitlynmichael) [reply content/operate/kubernetes/re-clusters/expand-pvc.md:95]: I've added this as a task here: https://redislabs.atlassian.net/browse/DOC-3754

File: content/operate/kubernetes/re-clusters/expand-pvc.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+{{<note>}}Shrinking (reducing the size) of your PVC is not allowed. This process only allows you to expand (size up) your PVC.{{</note>}}
+
+## Prerequisites
+
+{{<warning>}}Do not change any other REC fields related to the StatefulSet while resizing is in progress.
+{{</warning>}}
+
+- PVC expansion must be supported and enabled by the StorageClass and underlying storage driver of the REC PVCs.
+  - The relevant StorageClass is the one associated with the REC PVCs. The StorageClass for existing PVCs cannot be changed.
+- The StorageClass is must be configured with `allowVolumeExpansion: true`.
```
Comment (mich-elle-luna) [inline content/operate/kubernetes/re-clusters/expand-pvc.md:28]:
```suggestion
- The StorageClass must be configured with `allowVolumeExpansion: true`.
```

**Review by mich-elle-luna — APPROVED** [review]
thank you , just one small typo 'is must'

**Pull request merged**